docs_app/src/app/custom-elements/api/api-list.component.spec.ts

```diff
@@ -96,7 +96,7 @@ describe('ApiListComponent', () => {
     });
 
     it('should have no sections and no items visible when there is no match', () => {
-      component.setQuery('fizbuzz');
+      component.setQuery('fizzbuzz');
```

Comment:
Google Sheets suggested this change. I'm pretty sure it's a random playful word, but unless I have a good reason to fight Google, I let it win.